.github/workflows/ci.yml

```
@@ -2,34 +2,51 @@ name: CI
 on: [push, pull_request]
 jobs:
   test:
-    runs-on: ubuntu-18.04
+    runs-on: ubuntu-latest
     continue-on-error: ${{ matrix.flaky }}
+    name: "Tests: Ruby ${{ matrix.ruby-version }}"
+    if: "!contains(github.event.head_commit.message, '[skip ci]')"
     strategy:
       matrix:
-        ruby-version: ["2.5", "2.6", "2.7", "3.0", "3.1", "truffleruby-22", "truffleruby-21"]
+        ruby-version:
+          - 2.7
+          - 3.0
```

```suggestion
          - "3.0"
```

In order to avoid this YAML value rendering as the string "3".